src/garage/torch/embeddings/context_encoder.py

```diff
@@ -0,0 +1,15 @@
+# pylint: disable=too-few-public-methods
+"""Base context encoder class."""
+import abc
+
+
+class ContextEncoder(abc.ABC):
+    """Base class of context encoders for training meta-RL algorithms."""
+
+    def reset(self, num_tasks=1):
```

Comment:
Do you want to mark this as abstract? That will force implementers to implement it.